tests/test_aea_builder.py

```diff
@@ -668,13 +667,6 @@ def test_from_project(self):
         dummy_model = dummy_skill.models["dummy"]
         assert dummy_model.config == {"model_arg_1": 42, "model_arg_2": "2"}
 
-    def test_from_json(self):
-        """Test load project from json file with path specified."""
-        with open(Path(self._get_cwd(), DEFAULT_AEA_CONFIG_FILE), "r") as fp:
-            json_config = yaml_load_all(fp)
-
-        AEABuilder.from_config_json(json_config, Path(self._get_cwd()))
-
```

Comment:
I think we should keep this functionality. It is a good functionality to have.

Reply:
it was used only by my code i replaced